src/components/Displays/Display.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import '../Displays/display.css';
```

Comment:
This (and all your other CSS imports) can be simply `./display.css`

Also, standard practice is that the CSS file names match the .js file names exactly, so it should be `Display.css`

Additionally, though it has little to no effect on function, it's better practice to use the `.jsx` extension for any React components, instead of the `.js`
For clarification, a React component is any component that contains HTML in the return statement. So you can easily have `.js` and `.jsx` files in the same React app.